src/miner_lora_light.erl

```diff
@@ -0,0 +1,799 @@
+%%%-------------------------------------------------------------------
+%% @doc
+%% == Miner lora for light gateways ==
+%%          no use of chain or ledger
+%% @end
+%%%-------------------------------------------------------------------
+-module(miner_lora_light).
```

Comment:
Its probably worth reviewing this module in general has not diverged from miner_lora in any cases that mattered since there's definitely some miner_lora changes that went in after this module was forked off.

Reply:
yeah i have been doing that periodically.  Will do another pass